lib/ansible/modules/cloud/amazon/dynamodb.py

```diff
@@ -0,0 +1,359 @@
+    expression_attribute_values:
+        description:
+            - One or more values that can be substituted in an expressionself.
+              Use the ':' (colon) character in an expression to dereference
+              an attribute value.
+        required: false
+        default: null
+        type: dict
+requirements:
+    - boto3
+    - botocore
+extends_documentation_fragment:
+    - aws
+    - ec2
+'''
+
+EXAMPLES = '''
+# Returns a set of attributes for the item with the given primary key.
+- name: Gets a single item
+  dynamodb:
+    profile: pre
+    table: narcos
+    action: get
+    primary_key: {"bank": {"S": "hsbc"}}
+
+# Creates a single item in 'narcos' DynamoDB table where column 'bank' equals 'hsbc' string,
+# column 'quantity' equals '1000' numeric and column 'person' equals 'ochoa' string
+- name: Creates a new record
+  dynamodb:
+    table: narcos
+    action: put
+    item: {"bank": {"S": "hsbc"},"quantity": {"N": "1000"},"person": {"S": "ochoa"}}
+
+# Updates the 'quantity' attibute value from a single item.
+- name: Updates arribute 'person'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET quantity = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+
+# Updates the 'status' (dynamodb protected keyword) attibute value from a single item.
+- name: Updates arribute 'status'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}}
+    update_expression: 'SET #s = :number'
+    expression_attribute_values: {":number": {"N": "2000"}}
+    expression_attribute_names: {"#s" : "status"}
+
+# Deletes the 'person' attibute value from a single item. The table has a composite
+# primary key. 'bank' is the primary key and 'quantity' is the sort key.
+- name: Deletes arribute 'person'
+  dynamodb:
+    table: narcos
+    action: update
+    primary_key: {"bank": {"S": "hsbc"}, "quantity": {"N": "1000"}}
+    update_expression: 'REMOVE person'
+
+# Connects with awscli profile 'development' and
+# deletes a single item from 'releases' DynamoDB table
+# where the column 'project' equals string 'potatoes' (also primary key)
+# and column 'version' equals numeric '123456'
+- name: Delete a single item
+  dynamodb:
+    profile: development
+    table: releases
+    action: delete
+    primary_key: {"project": {"S": "potatoes"}}
+    condition_expression: version = :number
+    expression_attribute_values: {":number": {"N": "123456"}}
+'''
+
+RETURN = '''
+item:
+    description: Item when you peform a 'get' action.
+    returned: success
+    type: dict
+    sample:
+        {
+            "bank": {
+                "S": "hsbc"
+            },
+            "quantity": {
+                "N": "1000"
+            },
+            "person": {
+                "S": "ochoa"
+            }
+        }
+'''
+
+import ansible.module_utils.ec2
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
New modules should use AnsibleAWSModule, which provides a number of quality-of-life improvements over using AnsibleModule directly.

```suggestion
from ansible.module_utils.aws import AnsibleAWSModule
```